custom_components/mixergy/sensor.py

```diff
@@ -443,4 +461,4 @@ def icon(self):
 
     @property
     def name(self):
-        return f"Holiday Mode"
\ No newline at end of file
+        return f"Holiday Mode"
```

Comment:
Note that there was no new line at the end of this file. Even GitHub's online editing of the file wants to add one in.